test/0.1.2-rc.0/util.ts

```diff
@@ -69,6 +69,7 @@ export function getContractAddresses() {
     SchemeFactory: addresses.private.dao[arcVersion].Schemes[5].address,
     JoinAndQuit: addresses.private.dao[arcVersion].Schemes[6].address,
     FundingRequest: addresses.private.dao[arcVersion].Schemes[7].address,
+    // TokenTrade: addresses.private.dao[arcVersion].Schemes[8].address,
```

Comment:
You'll need to create the TokenTrade scheme in the test itself with `SchemeFactory` as it is not included in the test dao like other schemes (can't add so many schemes at the same time...). You should be able to do this fairly easily by following the example of the `SchemeFactory` test to see how to propose and pass in voting a proposal for a new scheme.

Reply:
All right, if I'm not wrong we are still getting the TokenTrade address from this function, but it will live inside the object `...addresses.private.package[arcVersion]` (like 60)